osmosis/README.md

```diff
@@ -104,10 +102,10 @@ export default () => {
 ```jsx
 //index.js Root Component
 import { StoreProvider } from '@shipt/osmosis';
-import { wrapCounter } from './counter.store';
+import { CounterStore } from './counter.store';
 import Counter from './counter';
 
-export default StoreProvider([wrapCounter], Counter);
+export default StoreProvider(CounterStore.Provider, Counter);
```

Comment:
the first argument for StoreProvider should be an array containing the store providers